algorithms/algorithm-practice-session-24.md

```diff
@@ -0,0 +1,8 @@
+# Strings 10
+
+<iframe src="https://adaacademy.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=2ffa060c-d25d-490d-9364-aec300055296&autoplay=false&offerviewer=true&showtitle=true&showbrand=true&captions=true&interactivity=all" height="405" width="720" style="border: 1px solid #464646;" allowfullscreen allow="autoplay"></iframe>
+
+### Resources
+
+* [Problems](https://docs.google.com/presentation/d/1eCM6dtbTJIqkkRlYHRcuqOWYciGJqsEIWyVrtAxLIik/edit?usp=sharing)
+* Solutions (coming soon)
```

Comment:
I realize this may affect all of the existing Algo Practice content in Learn, but have we considered changing "Solutions" to read "Sample Solutions" instead? I know in Thursdays at Ada we receive a lot of questions/concern around student implementations not matching the 'official' solution. 

Solutions slides may also benefit from a reminder that valid implementations may not match the proffered implementations, and that some sample student implementations can be observed in the included video recording. 
```suggestion
* Sample Solutions (coming soon)
```